### ISSUE TITLE
Can this library generate 128bits (16 bytes) or 160 bits (20 bytes) signature?

### ISSUE THREAD

**Issue by wwwossai** [Chia-Network/bls-signatures#315]
according to the original paper, BLS seems to support shortest signature size like 160bits, but i cant find any lib that can support that size directly. i know 128bits or 160bits is less secure, but produce more user friendly signature for keyboard typing, which can be used in some cases.

![image](https://user-images.githubusercontent.com/77321546/157809656-149204cb-2ef7-4c3b-a441-a74c13f837ce.png)

**Comment by mariano54** [comment]
No, we are using 96 byte signatures as defined in the BLS IETF specification: https://datatracker.ietf.org/doc/draft-irtf-cfrg-bls-signature/
